asmcomp/amd64/emit.mlp

```diff
@@ -615,15 +615,21 @@ let emit_instr fallthrough i =
         end;
         cfi_remember_state ();
         I.mov (domain_field Domainstate.Domain_c_stack) rsp;
-        (* FIXME: CFI *)
-        (* cfi_def_cfa_offset 8; *)
+        cfi_def_cfa_offset 8;
```

Comment:
Why is the offset 8 here? Do you need this? 8 doesn't look right.

Reply:
Since here the address will be determined by the value in memory, I think we should load the value into a register use `cfi_def_cfa` to set stack pointer to the register value